lib/resource-mapper.js

```diff
@@ -24,6 +25,7 @@ class ResourceMapper {
     this._defaultContentType = defaultContentType
     this._indexFilename = indexFilename
     this._types = { ...types, ...overrideTypes }
+    this._suffixes = fileSuffixes
```

Comment:
Idem. They're not just suffixes, they are suffixes for a specific kinds of files that needs special behavior.